images-list

```diff
@@ -64,6 +64,7 @@ gcr.io/google_containers/k8s-dns-sidecar rancher/mirrored-k8s-dns-sidecar 1.15.1
 gcr.io/google_containers/k8s-dns-sidecar rancher/mirrored-k8s-dns-sidecar 1.15.2
 gcr.io/kubebuilder/kube-rbac-proxy rancher/kube-rbac-proxy v0.5.0
 gcr.io/kubebuilder/kube-rbac-proxy rancher/mirrored-kube-rbac-proxy v0.5.0
+idealista/prom2teams rancher/idealista-prom2teams 3.2.1
```

Comment:
Both should use `mirrored-` prefix on image

Reply:
the prefix is appended.